Content/Metaballs/GrayGooMetaballs.cs

```diff
@@ -0,0 +1,97 @@
+﻿using StarlightRiver.Content.Items.Magnet;
```

Comment:
Should this file maybe be placed in the same directory as the item? Seems a bit strange to give this its own subdirectory in content when its used only for one item?

Reply:
It may be used for more items in the future, such as ferrofluid if it gets approved

Reply:
It might be a good idea to move this for now and move it back if it ends up re-used. Even if it is re-used for content in the same set I think a more appropriate place for this would be the Content/Items/Magnets folder if its only used for items in that set.